tasks/push-snapshot/push-snapshot.yaml

```diff
@@ -117,13 +117,11 @@ spec:
           oras_args=()
           if [[ "$media_type" == "application/vnd.docker.distribution.manifest.list.v2+json" ]]\
             || [[ "$media_type" == "application/vnd.oci.image.index.v1+json" ]]; then
-            # need to add separately or else the args get treated as 1 argument
-            # and oras complains about "Error: unknown flag: --platform linux/amd64"
-            oras_args+=("--platform")
-            oras_args+=("$os/$arch")
+            oras_args+=("--platform $os/$arch")
```

Comment:
To be honest I'm not even sure why a bash array is used here, especially since it's just one argument (or none).

How about just using a normal string here?
```sh
platform_arg=""
if ...
  platform_arg="--platform $os/$arch"
```

And then when you use it, it should be unquoted (same as with the array).

But if you want to stick with this since you already tested it, I'm fine with it.

Reply:
This was originally @ralphbean 's PR. I am unsure of whether he tried with a normal string?

Reply:
(I did a little experiment in bash to confirm that it works with a simple string.)

Reply:
let's keep it as-is since it is working currently.